vane/tests_tools.py

```diff
@@ -1501,6 +1513,98 @@ def transfer_file(self, src_file, dest_file, file_system, operation, dut=None, s
             pass
         return result
 
+    def run_snmpget(
+        self,
+        user,
+        auth_key,
+        priv_key,
+        oid,
+        dut=None,
+        auth_protocol=usmHMACSHAAuthProtocol,
+        priv_protocol=usmAesCfb256Protocol,
+    ):
```

Comment:
Hey Shachi might want to format the docstring according to the rules mentioned in the google space tasks section:

```suggestion
   """
        run_snmpget() runs a query against 'dut'
        using 'user', 'auth_key', 'priv_key' as cerdentials
        
        Args:
            user (type): snmp user
            auth_key (type): snmp auth key
            priv_key (type): snmp priv key
            oid (type): snmp oid
            dut (type): the dut against which snmp get needs to run
            auth_protocol (type): snmp auth protocol
            priv_protocol (type): snmp priv protocol
       """
```